tools/refresh-keyrings.sh

```diff
@@ -21,20 +21,24 @@ fi
 
 TARGET_DIR="$1"
 
-generate_keyring() {
-    KEYRING_FILE="$1"
+generate_keyrings() {
+    KEYRING_DIR="$1"
     shift
     KEYS="$@"
 
     # Intentionally unquoted so GPG gets the keys as separate arguments
-    if [ -e "$KEYRING_FILE" ]; then
-        mv "$KEYRING_FILE" "$KEYRING_FILE.old"
-    fi
-    gpg \
-        --keyring "$KEYRING_FILE" \
-        --no-default-keyring \
-        --keyserver keyserver.ubuntu.com \
-        --recv-keys $KEYS
+
+    for key in $KEYS; do
+	keyring_file=$KEYRING_DIR/ua-$key.gpg
+        if [ -e "$keyring_file" ]; then
+            mv "$keyring_file" "$keyring_file.old"
+        fi
+        gpg \
```

Comment:
Should we include `--no-options` in this invocation?

Reply:
sure let's